compiler/rustc_codegen_llvm/src/back/lto.rs

```diff
@@ -666,6 +667,38 @@ pub(crate) fn run_pass_manager(
     }
 
     if cfg!(llvm_enzyme) && enable_ad && !thin {
+        // This is the post-autodiff IR, mainly used for testing and educational purposes.
+        if config.autodiff.contains(&config::AutoDiff::PrintModAfter) {
```

Comment:
Did you pick that up from my experimental branch? I think we already handle PrintModAfter elsewhere, so this would print it twice. Can you check and potentially remove this line? Sorry, just noticed it now. Otherwise lgtm